### DISCUSSION TITLE
Html button in popperjs

### DISCUSSION THREAD

**Discussion by grztofdev** [floating-ui/floating-ui#1218] [Q&A]
Hi, 
why the Html button does not display in my popper?
Js looks like:
```js
    new Popover(item, {
        container: 'body',
        placement: 'bottom',
        html: true,
        title: 'elo',
        content: `<h1>Hi</h1><button>Hello</button>`
    })
```
The h1 tag renders good, but I can not see the button. Why

**Comment by FezVrasta** [comment]
You better ask to the correct repository, the example you posted is not Popper

**Reply by grztofdev** [reply]
@FezVrasta its from bootstrap. On plain popper js, buttons also do not render. Could you let me know why?

**Reply by FezVrasta** [reply]
I don't know, Popper doesn't render anything, it only applies some CSS
